Source/GSXibLoading.m

```diff
@@ -62,12 +63,40 @@ - (void) encodeWithCoder: (NSCoder *)coder
 
 - (id) initWithCoder: (NSCoder *)coder
 {
-  if([coder allowsKeyedCoding])
+  self = [super init];
+  if (self)
     {
-      [self setTypeIdentifier: [coder decodeObjectForKey: @"typeIdentifier"]];
-      [self setKeyPath: [coder decodeObjectForKey: @"keyPath"]];
-      [self setValue: [coder decodeObjectForKey: @"value"]];
+      if ([coder allowsKeyedCoding])
+        {
+          [self setTypeIdentifier: [coder decodeObjectForKey: @"typeIdentifier"]];
+          [self setKeyPath: [coder decodeObjectForKey: @"keyPath"]];
+          [self setValue: [coder decodeObjectForKey: @"value"]];
+
+          if ([coder containsValueForKey: @"type"])
```

Comment:
I would have preferred to solve this by adding an xml tag handler for userDefinedRuntimeAttribute in GSXib5KeyedUnarchiver.